### DISCUSSION TITLE
Drumroll editor's selection has wrong size

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#19809] [Q&A] [answered]
![2022 08 17-04:01:29](https://user-images.githubusercontent.com/53872073/185012594-0dc766d0-a4da-4f48-8be2-3c53235b2bfe.png)

Drumroll selection don't match the actual drumroll.
This also can be seen when editing exisiting map, for example, https://osu.ppy.sh/beatmapsets/1207412#taiko/2519222 (muzukashii)

![2022 08 17-04:10:58](https://user-images.githubusercontent.com/53872073/185012930-f6d7e1c6-7dad-4a9d-bb38-16755b045795.png)
)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/19810
